Aptoide/aptoide-client-v8
# [feature/ASV-1941] Home Skeleton

**What does this PR do?**

   This PR implements skeleton layout for initial home loading only, in order to improve home loading times. Note that most of it was done a month ago so review thoroughly.

**Database changed?**

   No

**How should this be manually tested?**

  Check if everything is working ok, i.e. skeleton appears and hides correctly; check that it only shows skeleton in the first loading and that every case is covered.

**What are the relevant tickets?**

  [ASV-1941](https://aptoide.atlassian.net/browse/ASV-1941)

**Code Review Checklist**
